packages/blocks/src/__internal__/rich-text/rich-text.ts

```diff
@@ -12,13 +12,10 @@ import { REFERENCE_NODE } from './reference-node.js';
 import { type AffineTextSchema, type AffineVEditor } from './virgo/types.js';
 
 const IGNORED_ATTRIBUTES = ['code', 'reference'] as const;
+export const REGEX_VALID_URL =
```

Comment:
There are existing url regex in

https://github.com/toeverything/blocksuite/blob/32475ade50ef1ee417452d949083dcd4f93c20ec/packages/blocks/src/components/link-popover/link-popover.ts#L29-L70